core/src/main/java/org/elasticsearch/search/suggest/completion/CompletionSuggestionBuilder.java

```diff
@@ -125,6 +122,26 @@ private CompletionSuggestionBuilder(String fieldname, CompletionSuggestionBuilde
     }
 
     /**
+     * Read from a stream.
+     */
+    public CompletionSuggestionBuilder(StreamInput in) throws IOException {
+        super(in);
+        payloadFields = new ArrayList<>();
```

Comment:
I tried to make it an array but it doesn't make the class simpler. ObjectParser like lists, as do the classes that interact with it.